roles/openshift_facts/defaults/main.yml

```diff
@@ -43,6 +43,7 @@ repoquery_installed: "{{ (ansible_pkg_mgr == 'dnf') | ternary('dnf repoquery --l
 openshift_use_crio: False
 openshift_use_crio_only: False
 openshift_crio_enable_docker_gc: True
+openshift_container_cli: "{{ openshift_use_crio | ternary('crictl', 'docker') }}"
```

Comment:
Need to cast openshift_use_crio to bool.

Reply:
Yeah, I forgot that my other change had not landed.
https://github.com/openshift/openshift-ansible/pull/9732/files#diff-3f7ba27815bc4fa8679d59a6e27e5d5cR136